docs/portal/transports/show-history.md

```diff
@@ -1,10 +1,10 @@
 # Show History of a Transport
 
-To see history of a transport, search the transport you want to show in the <strong> Transport </strong> page if it was not in the page, then click the "Action" button and select the "Show History".
-The <strong> Transports </strong> page can be reached simply by clicking "Transports" from the left pane of HDT portal.
+The <strong> Transports </strong> page can be reached by clicking "Transports" from the left pane of HDT portal.
+To view history of a transport, click the "Action" button of the transport entry in the list and select "Show History". If the transport was not in the current page, you can search it in the search box.
```

Comment:
Please insert the word "the" between "view" and "history."
Please change "search it" with "search for it."